pkg/kubectl/describe.go

```diff
@@ -830,7 +830,7 @@ func (d *PersistentVolumeDescriber) Describe(namespace, name string, describerSe
 		w := &PrefixWriter{out}
 		w.Write(LEVEL_0, "Name:\t%s\n", pv.Name)
 		printLabelsMultiline(w, "Labels", pv.Labels)
-		w.Write(LEVEL_0, "StorageClass:\t%s\n", storageutil.GetStorageClassAnnotation(pv.ObjectMeta))
```

Comment:
it is easily possible to add an annotation when converting from internal object to (externa) V1 one, so old applications (kubectl) will work.

There is one glitch though, I would need to mess up with serialization unit test (and probably fuzz tests too) for PVs and PVCs. The tests serialize bunch of weird API objects and they expect to get the same object after de-serialization. There is also internal -> v1 conversion done somewhere in the process that would add the annotation and therefore the input and output objects would differ. The code is scattered among several files, see e.g. Pod.Spec.InitContainers, they convert v1 beta annotation -> internal attribute in the same way as this PR:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unstructured_test.go#L48
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/conversion_test.go#L38
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/serialization_proto_test.go#L65
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/serialization_test.go#L387

I don't like this hacking of tests at all. Is strict backward compatibility with old kubectl necessary?